.github/workflows/beam_PerformanceTests_BigQueryIO_Batch_Java_Avro.yml

```diff
@@ -0,0 +1,93 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Performance Tests BigQueryIO Batch Java Avro
+
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    - cron: '0 */12 * * *'
```

Comment:
performance test uses 5 Dataflow workers at once. We have seen flakiness due to quota exceeded. The performance test result could also be impacted by the transiently available resources. Until GitHub Action support random schedule minute and hour, let's add some manual randomness on the schedule for performance test and load tests.

e.g. set the schedule to be sth like '10 1,13 * * *'; '30 8,20 * * *'

Reply:
Changed